manifests/init.pp

```diff
@@ -16,24 +16,37 @@
   $template = undef,
   $content = undef,
 ) {
-  if $::kernel == 'Linux' {
-    if $template {
-      if $content {
+  if $template {
+    if $content {
         warning('Both $template and $content parameters passed to motd, ignoring content')
       }
       $motd_content = template($template)
     }
-    elsif $content {
+      elsif $content {
       $motd_content = $content
     }
     else {
```

Comment:
This should move up to the previous line to be more in line with current best practices.